internal/cache/texture_desktop.go

```diff
@@ -1,12 +1,13 @@
-//go:build !android && !ios && !mobile
-// +build !android,!ios,!mobile
+//go:build !android && !ios && !mobile && !js && !wasm && !web
+// +build !android,!ios,!mobile,!js,!wasm,!web
 
 package cache
 
 // TextureType represents an uploaded GL texture
 type TextureType = uint32
 
-var noTexture = TextureType(0)
+// NoTexture used when there is no valid texture
+var NoTexture = TextureType(0)
```

Comment:
Does this need exposed? Noted below that it does not seem necessary to equate the cache and gl ZeroTextures

Reply:
From memory there are test that rely on NoTexture which make it more meaningful and easier to read.

Reply:
I think NoTexture is best being exposed as it is not always 0 (case for wasm). This allow code to not have to care about the compiler target.

Reply:
The new `Valid` exposed API seems to do the same thing? I don't think both have to be new public API do they?

Reply:
cache.NoTexture need to be a public API as with wasm it is not initialized to zero by default, but to undefined.

Reply:
Ugh, good point!